_data-prepper/common-use-cases/anomaly-detection.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: default
+title: Anomaly detection
+parent: Common use cases
+nav_order: 30
+---
+
+# Anomaly detection
+
+You can use Data Prepper to train models and generate anomalies in near real-time on time-series aggregated events. You can generate anomalies either on events generated within the pipeline, or on events coming directly into the pipeline, like OpenTelemetry metrics. You can feed these tumbling window aggregated time-series events to the [`anomaly_detector` processor]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/anomaly-detector/), which trains a model and generates anomalies with a grade score. Then write the anomalies to a separate index to create document monitors and trigger fast alerting.
```

Comment:
```suggestion
You can use Data Prepper to train models and generate anomalies in near real time on time-series aggregated events. You can generate anomalies either on events generated within the pipeline or on events coming directly into the pipeline, like OpenTelemetry metrics. You can feed these tumbling window aggregated time-series events to the [`anomaly_detector` processor]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/anomaly-detector/), which trains a model and generates anomalies with a grade score. Then write the anomalies to a separate index to create document monitors and trigger fast alerting.
```

Reply:
> Then write the anomalies to a separate index to create document monitors and trigger fast alerting.

The subject is unclear in my view. 

Maybe:

> You can configure your pipeline to write the anomalies to a separate index to create document monitors and trigger fast alerting.